dependencies/bdw-gc/meson.build

```diff
@@ -0,0 +1,22 @@
+# Boehm-gc Dependency File
+#============================================================================
+
+
+## FIXME !! Boehm is supposed to be an optional dependency
+## but is currently required for libexpr & libcmd to compile
+## correctly. 
```

Comment:
imo it should still be listed as a required build dependency, instead of optional. could trip up people trying to build nix :)

Reply:
I would say call it optional, because libcmd and libexr are *supposed* to build without them. If that's currently broken, it's not Meson's fault. If we give up on making them optional, that should be independent of this PR.

Reply:
yea exactly, not my place to be deciding on adding or removing the deps, this is just a buildsystem PR. but i thought id leave a comment just so its a known issue.

Reply:
i will add, that its ok detecting boehm in meson :) its just the docs dont list it. this was more of a pointer for docs/devs to look into this.